hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -48,6 +48,19 @@
       name:    "cs_aes_halt"
       act:     "req"
       package: "entropy_src_pkg"
+      desc:    '''
+               Coordinate activity between CSRNG's AES and Entropy Source's SHA3. The idea is that
+               Entropy Source requests CSRNG's AES to halt and waits for CSRNG to acknowledge before
+               it starts its SHA3. While SHA3 runs, Entropy Source keeps the request high. CSRNG may
+               not drop the acknowledge before Entropy Source drops the request.
+
+               Current limitations:
+               1. During startup and in Firmware Override - Extract & Insert mode, Entropy Source
+                  makes no AES Halt requests but still activates its SHA3 engine.
```

Comment:
This is okay because CSRNG's AES frequently stalls in this mode when waiting for entropy. SW has then to inject the entropy, trigger the SHA3 and only after that CSRNG can become active again and run the reseed/generate commands using AES.